Update Get-CsOnlineUser.md - Minor changes 2 (#9059)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Get-CsOnlineUser.md

```

### -Filter

> **Note: This immediate section only applies to TPM 2.6 and lower**

Reviewer: ```suggestion
**Note: This immediate section only applies to TPM 2.6 and earlier**
```

`{EnterpriseVoiceEnabled -eq $True}`

> The following updates are applicable for organizations having TeamsOnly users and/or without any SfBO/On-Premise users using Microsoft Teams PowerShell version 3.0.0 and above.

Reviewer: ```suggestion
The following updates are applicable for organizations having TeamsOnly users and/or without any SfBO/On-Premise users using Microsoft Teams PowerShell version 3.0.0 and later.
```

> The following updates are applicable for organizations having TeamsOnly users and/or without any SfBO/On-Premise users using Microsoft Teams PowerShell version 3.0.0 and above.

Reviewer: ```suggestion
The following updates are applicable for organizations having TeamsOnly users and/or without any SfBO/On-Premise users using Microsoft Teams PowerShell version 3.0.0 and later.
```

```

### -Identity

> **Note: This immediate section only applies to TPM 2.6 and lower**

Reviewer: ```suggestion
**Note: This immediate section only applies to TPM 2.6 and earlier**
```

## OUTPUTS

### Note:

> The following updates are applicable for organizations having TeamsOnly users and/or without any SfBO/On-Premise users using Microsoft Teams PowerShell version 3.0.0 and above.

Reviewer: ```suggestion
The following updates are applicable for organizations having TeamsOnly users and/or without any SfBO/On-Premise users using Microsoft Teams PowerShell version 3.0.0 and later.
```

1. LineURI set via OnPrem AD.
2. Direct Routing numbers assigned to users via Set-CsUser.

> In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via OnPrem AD and OnPremLineURIManuallySet attribute has been dropped. Direct Routing numbers will be available from the LineURI field. Direct Routing Numbers can be distinguished from Calling Plan Numbers by looking at the FeatureTypes attribute.

Reviewer 2: I would rephrase this to something along the lines of 'set via OnPrem AD. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used.'.

Reviewer 3: Agree

> In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via OnPrem AD and OnPremLineURIManuallySet attribute has been dropped. Direct Routing numbers will be available from the LineURI field. Direct Routing Numbers can be distinguished from Calling Plan Numbers by looking at the FeatureTypes attribute.

Reviewer 3: ```suggestion
In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via on-premises AD. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used. Direct Routing numbers will be available from the LineURI field. Direct Routing Numbers can be distinguished from Calling Plan Numbers by looking at the FeatureTypes attribute.
```

Reviewer 4: For consistency, please also change "Numbers" to "numbers" in the last sentence (two instances).

> In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via on-premises AD. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used. Direct Routing numbers will be available from the LineURI field. Direct Routing Numbers can be distinguished from Calling Plan Numbers by looking at the FeatureTypes attribute.

Reviewer 4: ```suggestion
In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via on-premises AD. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used. Direct Routing numbers will be available from the LineURI field. Direct Routing numbers can be distinguished from Calling Plan numbers by looking at the FeatureTypes attribute.
```

1. LineURI set via OnPrem AD.
2. Direct Routing numbers assigned to users via Set-CsUser.

> In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via on-premises AD. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used. Direct Routing numbers will be available from the LineURI field. Direct Routing numbers can be distinguished from Calling Plan numbers by looking at the FeatureTypes attribute.

Reviewer: ```suggestion
In Teams PowerShell Modules 3.0.0 and above OnPremLineURI will only refer to the LineURI set via on-premises Active Directory. The OnPremLineURIManuallySet attribute has been deprecated and is no longer used. Direct Routing numbers will be available from the LineURI field. Direct Routing numbers can be distinguished from Calling Plan numbers by looking at the FeatureTypes attribute.
```
